schema/int8_array.go

```diff
@@ -21,6 +21,10 @@ func (*Int8Array) Type() ValueType {
 	return TypeIntArray
 }
 
+func (dst *Int8Array) Size() int {
+	return 8 * len(dst.Elements)
```

Comment:
Can we call `Int8.Size()` here?

Reply:
I think this one is probably okay, the name even has `8` in it, and we're unlikely to redefine the size of 64-bit ints :) Otherwise we have to write:

```
if len(dst.Elements) == 0 {
     return 0
}
return dst.Elements[0] * len(dst.Elements)
```

And then there's still the assumption that all the elements have the same size. 

We could also do:

```
	totalSize := 0
	for _, element := range dst.Elements {
		totalSize += element.Size()
	}
	return totalSize
```

but that would be unnecessarily inefficient, being O(N) instead of O(1)